worlds/yachtdice/__init__.py

```diff
@@ -157,6 +158,19 @@ def generate_early(self):
             else:
                 self.itempool.append(cats[categorylist[index]])
 
+        # Give warnings and change start_inventory if necessary.
+        gave_warning = False
+        for item_name, count in self.multiworld.worlds[self.player].options.start_inventory.value.items():
+            if item_name.startswith("Category") and count > 20:
```

Comment:
Should be able to just do this:
```suggestion
        for item_name, count in self.options.start_inventory.value.items():
            if item_name.startswith("Category") and count > 20:
```